modules/preview-react/menu/lib/MenuItem.tsx

```diff
@@ -274,7 +274,7 @@ class MenuItem extends React.Component<MenuItemProps> {
           id={id}
           role={role}
           onClick={this.handleClick}
-          aria-disabled={!!isDisabled}
+          aria-disabled={isDisabled ? true : undefined}
```

Comment:
👍🏻

Reply:
We didn't catch this before, but this was actually a bug.